EIPS/eip-4747.md

```diff
@@ -245,15 +246,16 @@ On Ethereum Mainnet, the only state clearing events after the start of Byzantium
     0xfc4dc782bf7e81a2ed5cc0519f80de36e7931bd9
     0xfcde1c261eb257e14491b4e7cb1949a7623c00c5
     0xfd17a22fd80075f2716e93268aa01bcdd7d70b22
-    ```
-    </details>
+```
+
+</details>
```

Comment:
it seems you deleted the opening tag but retained closing tag. also not sure this change is needed but its upto the author to accept it. so I will wait till author's approval to give my approval